client/gutenberg/extensions/seo/editor.scss

```diff
@@ -0,0 +1,17 @@
+
+// @TODO: Replace with Gutenberg variables
+$dark-gray-500: #555d66;
+
+.jetpack-seo-message-box {
+	background-color: #edeff0;
```

Comment:
Was this supposed to be that `$dark-gray-500`?

Reply:
Nah, also copypasta from Publicize. This corresponds to `$light-gray-300`, so I've changed it to that for now.